src/ansys/workbench/core/workbench_client.py

```diff
@@ -381,6 +381,28 @@ def download_file(self, file_name, show_progress=True, target_dir=None):
             pbar.close()
         return file_name
 
+    def download_project_archive(self, archive_name, show_progress=True):
+        """Create and download the project archive.
+
+        Parameters
+        ----------
+        archive_name : str
+            The name of the project archive to use, without the file extension.
+        show_progress : bool, default: True
+            Whether to show a progress bar during the download.
+        """
+        if not re.match(r"^\w+$", archive_name):
+            logging.error("archive name contains illegal character")
```

Comment:
I suggest showing a clear message of what is the acceptable file name.

Reply:
Updated the message.